### PR TITLE
mongodb tests

### PR DESCRIPTION
**Pull request by madushan1000**

### CODE REVIEW

File: lib/modules/mongo/__tests__/index.js [hunk 1]
```diff
@@ -12,18 +18,68 @@ describe('module - mongo', function () {
   });
 
   describe('logs', function () {
-    it('TODO write tests');
+    it('should pull logs from "meteor" vm', async done => {
+      sh.cd(path.resolve(__rootdir, 'tests/project-1'));
+
+      sh.exec('mup mongo setup && mup mongo start', {silent: true});
+      const out = sh.exec('mup mongo logs', {silent: true});
```
Comment (arunoda) [inline lib/modules/mongo/__tests__/index.js]: Can be check for some log output. May be check some words which is common to MongoDB.

File: lib/modules/mongo/__tests__/index.js [hunk 2]
```diff
@@ -12,18 +18,68 @@ describe('module - mongo', function () {
...
+    });
   });
 
   describe('stop', function () {
-    it('TODO write tests');
+    it('should stop mongodb on "mongo" vm', async done => {
+
+      sh.cd(path.resolve(__rootdir, 'tests/project-1'));
+      sh.exec('mup mongo setup && mup mongo start', {silent: true});
+
```
Comment (arunoda) [inline lib/modules/mongo/__tests__/index.js]: Here can be check whether port 27017 is running.

File: lib/modules/mongo/__tests__/index.js [hunk 3]
```diff
@@ -12,18 +18,68 @@ describe('module - mongo', function () {
...
+      const out = sh.exec('mup mongo stop', {silent: true});
+      assert.equal(out.code, 0, 'mup mongo stop failed');
```
Comment (arunoda) [inline lib/modules/mongo/__tests__/index.js]: Check whether port 27017 is not running.

File: lib/modules/mongo/__tests__/index.js [hunk 4]
```diff
@@ -12,18 +18,68 @@ describe('module - mongo', function () {
...
+
+      sh.cd(path.resolve(__rootdir, 'tests/project-1'));
+      sh.exec('mup mongo setup', {silent: true});
+
+      const out = sh.exec('mup mongo start', {silent: true});
+      assert.equal(out.code, 0, 'mongo start failed');
+
+      assert.equal(countOccurences('start mongo: SUCCESS', out.output), 1, 'mongo start failed');
+
+      const sshOut = await runSSHCommand(serverInfo, 'sudo docker ps --format "{{ .Names }}"');
```
Comment (arunoda) [inline lib/modules/mongo/__tests__/index.js]: Also check for port 27017.

**Comment by arunoda** [comment]
@madushan1000 could you use `expect` instead of assert. 
See: https://github.com/kadirahq/lokka/blob/master/lib/__tests__/index.js#L10

**Comment by madushan1000** [comment]
Updated mongodb tests

**Pull request merged**